lib/rules/no-identical-tests.js

```diff
@@ -41,6 +41,14 @@ module.exports = {
                 context.report({
                   node: test,
                   message,
+                  fix (fixer) {
+                    const start = sourceCode.getTokenBefore(test);
+                    const end = sourceCode.getTokenAfter(test);
+                    return fixer.replaceTextRange(
+                      // should remove test's trailing comma
+                      [start.range[1], end.range[end.value === ',' ? 1 : 0]]
```

Comment:
I think this will handle whitespace differently depending on whether there is a comma. If there is no comma, then all the whitespace up to the next test will be removed. If there is a comma, the whitespace up to the next test will be preserved.

Maybe it would be better to be consistent about handling whitespace.